src/documents/migrations/1017_document_expired.py

```diff
@@ -0,0 +1,29 @@
+# Generated by Django 3.2.6 on 2022-02-27 12:09
+
+from django.db import migrations, models
+from django.db.migrations import RunPython
+from django_q.models import Schedule
+from django_q.tasks import schedule
+
+class Migration(migrations.Migration):
+
+    # pylint: disable=R0201
+    def add_schedules(apps, schema_editor):
+        schedule('documents.tasks.remove_expired_documents', name="Remove Expired Documents", schedule_type=Schedule.DAILY)
+
+    # pylint: disable=R0201
+    def remove_schedules(apps, schema_editor):
+        Schedule.objects.filter(func='documents.tasks.remove_expired_documents').delete()
+
+    dependencies = [
+        ('documents', '1016_auto_20210317_1351'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='document',
+            name='expired',
```

Comment:
`expired` sounds like a boolean. Kind of "is it expired or not?". I think the field should be called `expires` or `expires_at`, because it is nearly always a date in the future.